exercises/say/say_test.rb

```diff
@@ -1,128 +1,116 @@
-#!/usr/bin/env ruby
-gem 'minitest', '>= 5.0.0'
 require 'minitest/autorun'
 require_relative 'say'
 
+# Test data version: e3bd4a2
 class SayTest < Minitest::Test
-  def test_0
-    assert_equal 'zero', Say.new(0).in_english
+  def test_zero
+    # skip
+    question = '0'
+    assert_equal(zero, SayProblem.new(question).answer)
   end
 
   def test_one
     skip
-    assert_equal 'one', Say.new(1).in_english
+    question = '1'
+    assert_equal(one, SayProblem.new(question).answer)
```

Comment:
It looks like the assertions are different from the original.

In this case, its of the form:
```ruby
assert_equal expected, SayProblem.new(input).answer
```

But the original is:
```ruby
assert_equal expected, Say.new(input).in_english
```